autoblocks/_impl/tracer.py

```diff
@@ -125,14 +189,19 @@ async def _send_event_unsafe(
         trace_id = trace_id or self._trace_id
         timestamp = timestamp or datetime.now(timezone.utc).isoformat()
 
+        event = TracerEvent(
+            message=message,
+            trace_id=trace_id,
+            timestamp=timestamp,
+            properties=merged_properties,
+        )
+
+        transformed_event_json = await self._run_and_build_evals_properties(
+            evaluators, event, max_evaluator_concurrency
```

Comment:
nit: i like to use kwargs typically rather than positional args:

```python
transformed_event_json = await self._run_and_build_evals_properties(
  evaluators=evaluators,
  event=event,
  max_evaluator_concurrency=max_evaluator_concurrency,
)
```